### PR TITLE
Config update

### PR DESCRIPTION
**Pull request by xlvdk**
Update aws-okta to support `okta_aws_role_arn` so profiles can be
configured easier when using the credential process feature of the AWS
SDK.

In the AWS config `role_arn` requires either `source_profile` or
`credential_source` to be specified for the profile, otherwise it is
detected as partial credentials, ignoring the `credential_process`
field. Allow `aws-okta` to use the specifed role in the field
`okta_aws_role_arn` reduces the number of chained profiles required to
correctly use bare awscli, kubectl and other AWS SDK enabled

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,81 +20,47 @@ $ aws-okta add
...
+that your base role is one that has been configured for Okta's SAML integration
+by your Okta admin. Okta provides a guide for setting up that integration
+[here](https://support.okta.com/help/servlet/fileField?retURL=%2Fhelp%2Farticles%2FKnowledge_Article%2FAmazon-Web-Services-and-Okta-Integration-Guide&entityId=ka0F0000000MeyyIAC&field=File_Attachment__Body__s).
+During that configuration, your admin should be able to grab the AWS App Embed
+URL from the General tab of the AWS application in your Okta org.  You will need
+to set that value in your `~/.aws/config` file, for example:
 
 ```ini
-[okta]
+[profile okta]
```
Comment (Chippiewill) [inline README.md]: `[okta]` is a special TOML config block for okta specific defaults (like the saml URL). It's probably a bit confusing if you combine the `[okta]` block with the `[profile]` block and call the profile `okta` as well.

I'd at least rename this profile to something else.

File: README.md [hunk 2]
```diff
@@ -20,81 +20,47 @@ $ aws-okta add
...
 
-The `aws_saml_url` can be set in the "okta" ini section, or on a per profile basis. This is useful if, for example, your organization has several Okta Apps (i.e. one for dev/qa and one for prod, or one for internal use and one for integrations with third party providers). For example:
+The `aws_saml_url` can be set in the "okta" ini section, or on a per profile
+basis. This is useful if, for example, your organization has several Okta Apps
+(i.e. one for dev/qa and one for prod, or one for internal use and one for
+integrations with third party providers). For example:
 
 ```ini
-[okta]
+[profile okta-dev]
```
Comment (Chippiewill) [inline README.md]: ditto here

**Review by Chippiewill — APPROVED** [review]
minor comment, other lgtm

File: README.md (hunk 1 above)
Comment (xlvdk) [reply README.md]: I was wondering if it was something like that. I haven't used it before 😂  I think that is where some of my confusion came from.

File: README.md (hunk 2 above)
Comment (xlvdk) [reply README.md]: Yeah, I think I will need to give it a try specifying the [okta] part... Maybe this change isn't needed then 🤔

**Pull request merged**